src/app/games/coerceo/tests/CoerceoMove.spec.ts

```diff
@@ -66,13 +66,19 @@ describe('CoerceoMove', () => {
             expect(tileExchange.toString()).toBe('CoerceoMove((5, 5))');
             expect(deplacement.toString()).toBe('CoerceoMove((5, 5) > RIGHT > (7, 5))');
         });
-        it('CoerceoMove.encoder should be correct', () => {
-            const rules: CoerceoRules = new CoerceoRules(CoerceoPartSlice);
-            const minimax: CoerceoMinimax = new CoerceoMinimax(rules, 'CoerceoMinimax');
-            const moves: CoerceoMove[] = minimax.getListMoves(rules.node);
-            for (const move of moves) {
+        describe('encoder', () => {
+            it('should be correct with first turn moves', () => {
+                const rules: CoerceoRules = new CoerceoRules(CoerceoPartSlice);
+                const minimax: CoerceoMinimax = new CoerceoMinimax(rules, 'CoerceoMinimax');
+                const moves: CoerceoMove[] = minimax.getListMoves(rules.node);
+                for (const move of moves) {
+                    NumberEncoderTestUtils.expectToBeCorrect(CoerceoMove.encoder, move);
+                }
+            });
+            it('should be correct with tiles exchanges', () => {
+                const move: CoerceoMove = CoerceoMove.fromTilesExchange(new Coord(5, 7));
                 NumberEncoderTestUtils.expectToBeCorrect(CoerceoMove.encoder, move);
```

Comment:
C'quoi le bon X pour dire 
encode and decode are X since encode(value) = code and decode(code) = value.

Bijective ? expectToBeBijective :D ?
expectToBeCorrect me fait toujours tilter :D "expectToBeGentil" :D

Reply:
bijective se dit d'une fonction f(x) qui pour chaque x donne un y différent (donc f(x1) = f(x2) => x1 = x2, i.e., elle est injective), et qui a un y pour chaque x (i.e., est surjective). Donc ici pas grand chose à voir, car on n'est du tout sur le même niveau de raisonnement : c'est *une* fonction injective, ici on parle d'une paire de fonction.
Le seul autre terme que "correct", serait : decode est l'inverse d'encode et vice-versa. Mais du coup `expectEncodeAndDecodeToBeInverse` ?

Reply:
Oui je propose ça (on peut aussi ticketter ça pour faire ça à un autre moment) on trouvera peut être un nom plus sympa plus tard.

Reply:
enfin je soutient ta proposition plutôt

Reply:
ok j'ajoute au ticket renommage !